shithead_game_server/src/lobby.rs

```diff
@@ -233,11 +394,59 @@ impl Lobby {
                 .expect("not enough cards to initialize game")
                 .collect();
         }
+
+        // start the first turn
+        let first_turn_player_id = self.player_list.first_turn_player();
+        self.set_current_turn_and_update_players(first_turn_player_id);
     }
 
     /// Returns the lobby player with the given id.
     pub fn get_player(&self, player_id: ClientId) -> &LobbyPlayer {
-        &self.players[&player_id]
+        &self.player_list[player_id]
+    }
+
+    /// Finds the id of the player who will play in the next turn.
+    fn next_turn_player_id(&self) -> ClientId {
+        match &self.current_turn {
+            Some(current_turn) => self
+                .player_list
+                .find_next_turn_player_after(current_turn.player_id),
+            None => self.player_list.first_turn_player(),
+        }
+    }
+
+    /// The player has finished playing his turn, move on the next turn.
+    pub async fn turn_finished(&mut self) {
```

Comment:
Add a comment that tit'll be used later when the `clickedCard` message will have handling. Because until now I understood that the player will always take the trash.